### PR TITLE
Added support for passing transaction/connection into EventStore

### PR DESCRIPTION
**Pull request by bolicd**

### CODE REVIEW

File: Tacta.EventStore/Repository/EventStoreRepository.cs [hunk 1]
```diff
@@ -11,6 +12,8 @@ namespace Tacta.EventStore.Repository
     public sealed class EventStoreRepository : IEventStoreRepository
     {
         private readonly ISqlConnectionFactory _sqlConnectionFactory;
+        private readonly IDbConnection _connection = null;
```
Comment (nusretasinanovic) [inline Tacta.EventStore/Repository/EventStoreRepository.cs:15]: Assigned but never used?

File: Tacta.EventStore/Repository/EventStoreRepository.cs [hunk 2]
```diff
@@ -11,6 +12,8 @@ namespace Tacta.EventStore.Repository
...
+        private readonly IDbTransaction _transaction = null;
```
Comment (nusretasinanovic) [inline Tacta.EventStore/Repository/EventStoreRepository.cs:16]: Always null, but used in get methods?

**Review by nusretasinanovic — APPROVED** [review]
Apart from unused variables and weird spacing, looks good

File: Tacta.EventStore/Repository/EventStoreRepository.cs (hunk 2 above)
Comment (bolicd) [reply Tacta.EventStore/Repository/EventStoreRepository.cs]: removed

File: Tacta.EventStore/Repository/EventStoreRepository.cs (hunk 1 above)
Comment (bolicd) [reply Tacta.EventStore/Repository/EventStoreRepository.cs]: removed

**Pull request merged**